### ISSUE TITLE
Questions about dataset splitting

### ISSUE THREAD

**Issue by chihina** [dk-kim/DFWSGAR#10]
Thanks for your great research!
In these codes, where dataset splitting (e.g., train, valid, and test) is operated?

**Comment by dk-kim** [comment]
Thank you for your attention to our work. 
It is operated on dataloader/dataloder.py.

**Comment by chihina** [comment]
Thanks for your response.

I checked the dataloader/dataloder.py.
If my understanding is correct, the codes use test data as validation data.
In line 158 of train.py, the model which can achieve the best results on the test data is saved?

**Comment by dk-kim** [comment]
Your understanding is correct. We follow previous GAR methods (SSU, ARG, etc.) for dataset split and evaluation protocol.

**Comment by chihina** [comment]
I considered the evaluation protocol not good because the model is optimized for test data.
The original evaluation protocol of the Volleyball dataset is shown at https://github.com/mostafa-saad/deep-activity-rec.

**Comment by dk-kim** [comment]
I agree with your opinion. Evaluation protocol for current GAR methods measure more like validation accuracy. 
You may refer these references. 
(SSU: https://github.com/cvlab-epfl/social-scene-understanding, 
ARG: https://github.com/wjchaoGit/Group-Activity-Recognition,
DIN: https://github.com/JacobYuan7/DIN-Group-Activity-Recognition-Benchmark)

If you have further discussions about our paper not for the code, please send email to me.
